app.js

```diff
@@ -38,6 +38,25 @@ app.get('/ping', (req, res) => {
     res.status(200).json({ message: 'pong' });
 });
 
+//// Sign up
+app.post('/signUp', async (req, res) => {
+    const { name, email, password } = req.body;
+    const saltRounds = 12;
+    const hashPassword = await bcrypt.hash(password, saltRounds);
+    console.log(hashPassword);
+    const result = await myDataSource.query(
+        `
+        INSERT INTO users(
+        name,
+        email,
+        password
+        ) VALUES (?, ?, ?);
+        `,
+        [name, email, hashPassword]
+    );
+    res.status(201).json({ message: 'user Created' });
+});
+
```

Comment:
비슷한 작업이 아닌 경우 줄바꿈을 통해 코드를 분리하고 
모든 코드에 들여쓰기를 맞춰주는 것이 가독성이 좋습니다.
```suggestion
//// Sign up
app.post('/signUp', async (req, res) => {
    const { name, email, password } = req.body;
    
    const saltRounds = 12;
    const hashPassword = await bcrypt.hash(password, saltRounds);

    const result = await myDataSource.query(
        `
            INSERT INTO users (
                name,
                email,
                password
            ) VALUES (
                ?, 
                ?, 
                ?
            );
        `,
        [name, email, hashPassword]
    );
    res.status(201).json({ message: 'user Created' });
});

```

Reply:
네 처음에 git에 대해 설명을 들을 때 키보드 조작 중에 잘못 기입된 부분도 있고 빠진 부분을 확인해서
처음부터 차근차근 다시 제가 해보자 라는 마음으로 해봤습니다.